doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -185,6 +196,93 @@
     "plt.colorbar(label='RMSD (Angstrom)')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pairwise RMSD between two trajectories"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calculating the 2D RMSD between two trajectories is a bit more finicky; `DistanceMatrix` can only calculate the RMSD of a trajectory to itself. Instead, we do it the long way by simply calculating the RMSD of each frame in the second trajectory, to each frame in the first trajectory.\n",
```

Comment:
I know it's rather obvious, but assuming someone that's only just getting started in the field, maybe we want to make a point to say that this will only work for equal length trajectories.

Reply:
I don't think it has to, as we just calculate each frame to every other frame instead of elementwise?

Reply:
Yes! My bad, I completely misinterpreted what was going on with that loop.